CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @kubawach
```

Comment:
I'd add at least one more Java folk (eg @mateuszrzeszutek or @johnbley) or sbd who can redistribute the review request if no code owner is available (@iNikem or @flands).

Reply:
PRs are welcomed! :)

Reply:
You can add me, I've dabbled with this repo before.